support-frontend/assets/components/subscriptionCheckouts/address/deliveryCheck.js

```diff
@@ -0,0 +1,45 @@
+const M25_POSTCODE_PREFIXES = [
```

Comment:
here is the list from subs frontend
https://github.com/guardian/subscriptions-frontend/blob/0f01397786041b2b225e73bf764c8732f41c0948/assets/javascripts/modules/checkout/deliveryDetails.js#L25

Reply:
There are some postcodes missing in the list here that are in the list from subs:
```
[ 'CM14',
  'DA6',
  'EC1',
  'EC2',
  'EC3',
  'EC4',
  'KT1',
  'N9',
  'RM5',
  'RM9',
  'WD2' ]
```

Reply:
And there are also loads more in your list that aren't included in the subs list.

Reply:
the list from subs frontend is the wrong list, I tried to attach the correct list here but it didn't work, I can share it separately

Reply:
👍

Reply:
I see, I wonder wrong list in terms of what? did the list come from cex team or did you just go by the "M25" defintion?

Reply:
the list i received from Matthew Sullivan that he got from Newsprinters (the delivery company)

Reply:
great thanks in that case it will be perfect 👍